_rules/SC2-1-2-no-keyboard-trap.md

```diff
@@ -0,0 +1,241 @@
+---
+name: No keyboard trap
+rule_type: composite
+description: |
+  This rule checks if it is possible to use standard keyboard navigation to navigate through all content on a web page without becoming trapped in any element, or that it is possible to use non-standard keyboard navigation to navigate through content where focus is trapped when using standard ways of keyboard navigation.
```

Comment:
Loooong sentence. How about (just an idea):

[[
This rule checks for keyboard traps. This includes use of standard keyboard navigation to navigate through all content without becoming trapped, and information about non-standard keyboard navigation when they are necessary.
]]

Reply:
Good suggestion. However, the user should be able to use non-standard keyboard navigation, and not only be informed that it exists.

Updated suggestion:
[[
This rule checks for keyboard traps. This includes use of both standard and non-standard keyboard navigation to navigate through all content without becoming trapped.
]]

Reply:
works for me! Very succinct.